CHANGELOG/CHANGELOG-0.9.md

```diff
@@ -26,6 +26,33 @@ Changes since `v0.8.0`:
 
   The QueueVisibility feature and its corresponding API was deprecated and will be removed in the v1beta2. Please use VisibilityOnDemand (https://kueue.sigs.k8s.io/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand/) instead. (#3110, @mbobrovskyi)
 
+## How to upgrade Kueue to 0.9.x
```

Comment:
```suggestion
## Upgrading steps
```
nit, it feels a bit weird to start with "How" and not end with "?". Also Kueue is redundant here.

Reply:
Done